lib/Elastica/Client.php

```diff
@@ -669,48 +684,23 @@ public function refreshAll()
     }
 
     /**
-     * logging.
-     *
-     * @param string|\Elastica\Request $context
-     *
-     * @throws Exception\RuntimeException
-     */
-    protected function _log($context)
```

Comment:
I've added _log back with similar behaviour above. It's still not perfect: without adding a second parameter (which will cause php errors if someone has overwritten _log) we cant get the Response for logging details about the response.

I think its an okay trade-off, its simple to fix by adding the second parameter to the overwritten _log method.